src/components/network/network-panel.js

```diff
@@ -45,20 +45,30 @@ genotet.NetworkPanel.prototype.initPanel = function() {
         }.bind(this));
   }, this);
 
+  // Input type update
+  this.container.find('#regex input').click(function() {
+    this.isRegex_ = true;
+  }.bind(this));
+  this.container.find('#string input').click(function() {
+    this.isRegex_ = false;
+  }.bind(this));
+  this.container.find('#regex input').trigger('click');
```

Comment:
You need to understand every bit of code you wrote and copied. Read more about jquery and its event handling may help you.